app/controllers/bank_accounts_controller.rb

```diff
@@ -22,15 +28,27 @@ def create
     @bank_account = current_organization.bank_accounts.build(bank_account_params)
 
     if @bank_account.save
-      redirect_to bank_accounts_path, notice: 'Bank account was successfully created.'
+      respond_to do |format|
+        format.html do
+          redirect_to bank_accounts_path, notice: 'Bank account
```

Comment:
@dissident and don't make new line in middle of string. We don't require to be in 80 columns per line. Or if you want to make on new line the concatenate strings `"..." + "..."`